modules.d/99openssl/module-setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+check() {
+    return 255
+}
+
+install() {
+    local ossl_files="${dracutbasedir}/ossl-files"
+
+    local openssl_cnf="$($ossl_files --config)"
+
+    local initrd_openssl_cnf="${initdir}/${openssl_cnf}"
+
+    if [[ ! -r "$openssl_cnf" ]]; then
+        dfatal "'$ossl_files --config' does not return a path!!"
+        exit 1
+    fi
+
+    inst_multiple -o \
+      /etc/pki/tls/openssl.cnf \
+      /etc/pki/tls/openssl.d/* \
+      /etc/crypto-policies/back-ends/opensslcnf.config \
+      /usr/lib64/ossl-modules/*.so \
```

Comment:
Technically redundant, since the line after that will contain the same files. If we're aiming for the minimal initramfs, we should probably remove this line to save some space.

Reply:
Yes, exactly my thoughts :). Thanks for confirmation.